arrays/arrays.c

```diff
@@ -73,7 +73,7 @@ void resize_array(Array *arr)
 char *arr_read(Array *arr, int index)
 {
 
-  // Throw an error if the index is greater than the current count
+  // Throw an error if the index is greater than or equal to the current count
 
```

Comment:
lines 31-2
```
// Allocate memory for elements
arr->elements = malloc(sizeof(char *) * capacity);
```

you do not want to use malloc per the test ` mu_assert(arr_read(arr, 0) == NULL, "Value initialized not null");`

Reply:
Brady gave that code to us in lecture yesterday. . .